src/Symfony/Component/Cache/Traits/PdoTrait.php

```diff
@@ -373,19 +373,24 @@ private function getConnection()
                     case $driver instanceof \Doctrine\DBAL\Driver\DrizzlePDOMySql\Driver:
                     case $driver instanceof \Doctrine\DBAL\Driver\Mysqli\Driver:
                     case $driver instanceof \Doctrine\DBAL\Driver\PDOMySql\Driver:
+                    case $driver instanceof \Doctrine\DBAL\Driver\PDO\MySQL\Driver:
                         $this->driver = 'mysql';
                         break;
                     case $driver instanceof \Doctrine\DBAL\Driver\PDOSqlite\Driver:
+                    case $driver instanceof \Doctrine\DBAL\Driver\PDO\SQLite\Driver:
                         $this->driver = 'sqlite';
                         break;
                     case $driver instanceof \Doctrine\DBAL\Driver\PDOPgSql\Driver:
+                    case $driver instanceof \Doctrine\DBAL\Driver\PDO\PgSQL\Driver:
                         $this->driver = 'pgsql';
                         break;
                     case $driver instanceof \Doctrine\DBAL\Driver\OCI8\Driver:
                     case $driver instanceof \Doctrine\DBAL\Driver\PDOOracle\Driver:
+                    case $driver instanceof \Doctrine\DBAL\Driver\PDO\OCI\Driver:
                         $this->driver = 'oci';
                         break;
                     case $driver instanceof \Doctrine\DBAL\Driver\SQLSrv\Driver:
+                    case $driver instanceof \Doctrine\DBAL\Driver\PDO\SQLSrv\Driver:
```

Comment:
Not sure about this one. `Doctrine\DBAL\Driver\PDOSqlsrv\Driver` was renamed to `Doctrine\DBAL\Driver\PDO\SQLSrv\Driver` (but not `Doctrine\DBAL\Driver\SQLSrv\Driver`).